apps/vr-tests/src/stories/react-charting/LineChart.stories.tsx

```diff
@@ -0,0 +1,967 @@
+                endIndex: 2,
+              },
+            ],
+            data: [
+              {
+                x: new Date('2020-03-03T10:30:00.000Z'),
+                y: 218123,
+              },
+              {
+                x: new Date('2020-03-03T11:00:00.000Z'),
+                y: 219000,
+              },
+              // gap here
+              {
+                x: new Date('2020-03-04T00:00:00.000Z'),
+                y: 248000,
+              },
+              {
+                x: new Date('2020-03-05T00:00:00.000Z'),
+                y: 252000,
+              },
+              // gap here
+              {
+                x: new Date('2020-03-07T00:00:00.000Z'),
+                y: 260000,
+              },
+              {
+                x: new Date('2020-03-08T00:00:00.000Z'),
+                y: 300000,
+              },
+            ],
+            color: DefaultPalette.blue,
+          },
+          {
+            legend: 'Green Data',
+            lineOptions: {
+              lineBorderWidth: '4',
+            },
+            data: [
+              {
+                x: new Date('2020-03-03T00:00:00.000Z'),
+                y: 297000,
+              },
+              {
+                x: new Date('2020-03-04T00:00:00.000Z'),
+                y: 284000,
+              },
+              {
+                x: new Date('2020-03-05T00:00:00.000Z'),
+                y: 282000,
+              },
+              {
+                x: new Date('2020-03-06T00:00:00.000Z'),
+                y: 294000,
+              },
+              {
+                x: new Date('2020-03-07T00:00:00.000Z'),
+                y: 224000,
+              },
+              {
+                x: new Date('2020-03-08T00:00:00.000Z'),
+                y: 300000,
+              },
+              {
+                x: new Date('2020-03-09T00:00:00.000Z'),
+                y: 298000,
+              },
+              {
+                x: new Date('2020-03-10T00:00:00.000Z'),
+                y: 299000,
+              },
+            ],
+            color: DefaultPalette.green,
+          },
+        ],
+      };
+
+      const rootStyle = { width: `${700}px`, height: `${300}px` };
+      const margins = { left: 35, top: 20, bottom: 35, right: 20 };
+
+      return (
+        <>
+          <div style={rootStyle}>
+            <LineChart
+              data={data}
+              legendsOverflowText={'Overflow Items'}
+              yMinValue={150000}
+              yMaxValue={400000}
+              height={300}
+              width={700}
+              margins={margins}
+              calloutProps={{
+                calloutMaxWidth: 200,
+              }}
+              getCalloutDescriptionMessage={_calculateCalloutDescription}
+              enablePerfOptimization={true}
+            />
+          </div>
+        </>
+      );
+    },
+    { includeRtl: true, includeDarkMode: true },
+  )
+  .addStory(
+    'Date_Axis',
```

Comment:
We can remove date axis story and add story to test - hover card for single line.